### PR TITLE
Jacques/docker compose docs

### PR DESCRIPTION
**Pull request by jverre**
## Details

Updated the Opik documentation with the new self-hosting instructions using the docker-compose approach rather than the Opik installer.

Before this PR is merged, we will need to merge the docker compose PR and the `opik login` PR.

### CODE REVIEW

**Comment by liyaka** [comment]
@jverre please see the changes in my PR for docker-compose - moved to separate folder and added README

**Comment by jverre** [comment]
Thanks @liyaka, yep once the other PR is merged I'll update the doc

File: docker-compose.yaml
Comment (andrescrz) [inline docker-compose.yaml:1]: Overlaps: https://github.com/comet-ml/opik/pull/245/files

File: nginx_default_local.conf
Comment (andrescrz) [inline nginx_default_local.conf:1]: Overlaps: https://github.com/comet-ml/opik/pull/245/files

File: sdks/python/src/opik/cli.py
Comment (andrescrz) [inline sdks/python/src/opik/cli.py:1]: Please double check changes in this file. CC: @alexkuzmik

**Review by andrescrz — CHANGES_REQUESTED** [review]
Please see overlapping changes.

**Pull request merged**